packages/contracts-bedrock/test/fixtures/test-deploy-superchain-out.toml

```diff
@@ -1,3 +1,16 @@
+[dsi]
```

Comment:
IMO this naming convention makes sense in the code, but is a bit unclear for the toml files, and I expect that many more people will read the config files than the deploy scripts. I'd suggest something like
 
```suggestion
[superchain]
```

Reply:
Actually, I think my suggestion will put all outputs under the roles header, so maybe we put them under a `contracts` or `outputs` header. The outputs from all steps are always contract addresses

Reply:
> Actually, I think my suggestion will put all outputs under the roles key
It will yea. 

Good to go with `[outputs]` though.

Reply:
If we didn't have the comments, I'd probably be pushing for some identifier/label to indicate which step of the process the output file is from. But the comments make it clear.

Reply:
> Personally, if I've looking at an output file. 

In general I agree, however I think these output files are more likely to be consumed by other tools (op-deployer, superchain registry) than used directly by humans as a reference. We should still aim to make them as clear and readable as we can though, but for example when using the DeploySuperchain outfile to configure a genesis file, it doesn't matter that `proxyAdminOwner` was an input to `DeploySuperchain.s.sol`, it's still an input for genesis generation regardless

Reply:
Added in: https://github.com/ethereum-optimism/optimism/pull/11975/commits/d11660e77476e00456829dc3a37fb499d65328e9#diff-62b3d04f126e442fa8b92be128f8e5a03d3470e09d65f5f5657c5d0f842eff90R204 

I'm not using `awk` as discussed but instead using `printf` and `cat` both of which behave the same across platform. I personally find it easier to read.